src/sdg/api/tests/test_api.py

```diff
@@ -103,3 +106,296 @@ def test_retrieve_location_by_uuid(self):
+        )
+        lokale_overheid = LokaleOverheidFactory.create(organisatie=organisatie)
+        locatie = LocatieFactory.create(lokale_overheid=lokale_overheid)
+        token_authorization = TokenAuthorizationFactory.create(
+            lokale_overheid=lokale_overheid
+        )
+
+        detail_url = reverse("api:locatie-detail", args=[locatie.uuid])
+
+        body = {
+            "naam": "Lorem Ipsum",
+            "straat": "Lorem Ipsum",
+            "nummer": "12",
+            "postcode": "1234AB",
+            "plaats": "Lorem Ipsum",
+            "land": "Lorem Ipsum",
+            "openingstijden": {
+                "maandag": ["12:00 - 18:00"],
+                "dinsdag": ["12:00 - 18:00"],
+                "woensdag": ["12:00 - 18:00"],
+                "donderdag": ["12:00 - 18:00"],
+                "vrijdag": ["12:00 - 18:00"],
+                "zaterdag": ["12:00 - 18:00"],
+                "zondag": ["12:00 - 18:00"],
+            },
+            "openingstijdenOpmerking": "Lorem Ipsum",
+            "organisatie": {
+                "owmsIdentifier": organisatie.owms_identifier,
+                "owmsPrefLabel": organisatie.owms_pref_label,
+                "owmsEndDate": organisatie.owms_end_date,
+            },
+        }
+
+        headers = {"HTTP_AUTHORIZATION": f"Token {token_authorization.token}"}
+
+        response = self.client.put(
+            detail_url,
+            data=json.dumps(body),
+            content_type="application/json",
+            **headers,
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        data = response.json()
+
+        self.assertEqual(str(locatie.uuid), data["uuid"])
+        self.assertEqual(data["naam"], "Lorem Ipsum")
+        self.assertEqual(data["straat"], "Lorem Ipsum")
+        self.assertEqual(data["nummer"], "12")
+        self.assertEqual(data["postcode"], "1234AB")
+        self.assertEqual(data["plaats"], "Lorem Ipsum")
+        self.assertEqual(data["naam"], "Lorem Ipsum")
+        self.assertEqual(data["openingstijden"]["maandag"], ["12:00 - 18:00"])
+        self.assertEqual(data["openingstijden"]["dinsdag"], ["12:00 - 18:00"])
+        self.assertEqual(data["openingstijden"]["woensdag"], ["12:00 - 18:00"])
+        self.assertEqual(data["openingstijden"]["donderdag"], ["12:00 - 18:00"])
+        self.assertEqual(data["openingstijden"]["vrijdag"], ["12:00 - 18:00"])
+        self.assertEqual(data["openingstijden"]["zaterdag"], ["12:00 - 18:00"])
+        self.assertEqual(data["openingstijden"]["zondag"], ["12:00 - 18:00"])
+        self.assertEqual(data["openingstijdenOpmerking"], "Lorem Ipsum")
+        self.assertEqual(
+            data["organisatie"]["owmsIdentifier"],
+            "http://standaarden.overheid.nl/owms/terms/test",
+        )
+        self.assertEqual(data["organisatie"]["owmsPrefLabel"], "test")
+        self.assertEqual(data["organisatie"]["owmsEndDate"], None)
+
+    def test_delete_location(self):
```

Comment:
Can you add a test to delete a location that the token does not have permission to?